<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtCefClass</class>
 <widget class="QMainWindow" name="QtCefClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtCef</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_zoom_in">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ZoomIn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_zoom_out">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ZoomOut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_newwin">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>NewWin</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtcef.qrc"/>
 </resources>
 <connections/>
</ui>
